Dockerfile.invocation-image

```diff
@@ -0,0 +1,21 @@
+ARG ALPINE_VERSION=3.8
+
+FROM dockercore/golang-cross:1.11.5@sha256:17a7e0f158521c50316a0d0c1ab1f6a75350b4d82e7ef03c98bcfbdf04feb4f3 AS build
```

Comment:
The idea of using ARGs for `ALPINE_VERSION` and `GO_VERSION` was to ensure that we used the same versions across the various Dockerfiles.

I think it would make sense to use an ARG for `dockercore/golang-cross` that we maintain from a centralised point (`vars.mk`?).

Reply:
This part is only to build the docker/cli master and will be removed at the next 19.03 release.